packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -27,7 +27,7 @@
     "ajv": "^6.1.1",
     "eslint": "4.16.0",
     "eslint-plugin-lwc": "0.4.0",
-    "lwc-language-server": "1.6.3",
+    "lwc-language-server": "1.6.4",
```

Comment:
Not sure if lwc-next should be on another version given it's a preview of what's coming in lwc.